src/mocks/fakeData.ts

```diff
@@ -0,0 +1,891 @@
+import { LocationInfoType } from 'types/cityInfoType';
+
+export const fakeSortedCities: LocationInfoType[] = [
+  {
+    id: 105128581,
+    name: 'New York',
+    country: 'United States',
+    timezone: 'America/New_York',
+    language: 'en',
+    adminArea: 'New York',
+    adminArea2: '',
+    adminArea3: '',
+    lon: -74.006,
+    lat: 40.7143
+  }
+];
+
+export const fakeFavoriteCities = [
+  {
+    id: 105128581,
+    name: 'New York',
+    country: 'United States',
+    timezone: 'America/New_York',
+    language: 'en',
+    adminArea: 'New York',
+    adminArea2: '',
+    adminArea3: '',
+    lon: -74.006,
+    lat: 40.7143
+  },
+  {
+    id: 102634715,
+    name: 'Washington',
+    country: 'United Kingdom',
+    timezone: 'Europe/London',
+    language: 'en',
+    adminArea: 'England',
+    adminArea2: 'Sunderland',
+    adminArea3: null,
+    lon: -1.5167,
+    lat: 54.9
+  },
+  {
+    id: 100625144,
+    name: 'Minsk',
+    country: 'Belarus',
+    timezone: 'Europe/Minsk',
+    language: 'en',
+    adminArea: 'Minsk',
+    adminArea2: null,
+    adminArea3: null,
+    lon: 27.5667,
+    lat: 53.9
+  },
+  {
+    id: 100625144,
+    name: 'Minsk',
+    country: 'Belarus',
+    timezone: 'Europe/Minsk',
+    language: 'en',
+    adminArea: 'Minsk',
+    adminArea2: null,
+    adminArea3: null,
+    lon: 27.5667,
+    lat: 53.9
+  }
+];
+
+export const fakeCurrentWeather = {
+  current: {
+    time: '2022-10-21T10:03+03:00',
+    symbol: 'd400',
+    symbolPhrase: 'overcast',
+    temperature: 7,
+    feelsLikeTemp: 6,
+    relHumidity: 89,
+    dewPoint: 5,
+    windSpeed: 2,
+    windDir: 225,
+    windDirString: 'SW',
+    windGust: 5,
+    precipProb: 3,
+    precipRate: 0,
+    cloudiness: 100,
+    thunderProb: 0,
+    uvIndex: 0,
+    pressure: 1024.62,
+    visibility: 11243
+  }
+};
+
+export const fakeFeedbacks = [
```

Comment:
i'd recommend to add type definitions to all mocked data so in case your types will be updated in future, you could easily update mock data